<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmFittingBox</class>
 <widget class="QWidget" name="frmFittingBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>180</width>
    <height>120</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>180</width>
    <height>120</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>180</width>
    <height>120</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="line_vertical">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0.258</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>56</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>merge Similar Plane</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>82</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Parallal thresh.</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_parall">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>7</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0.966</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_merge">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>54</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0.2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>33</y>
     <width>82</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Vertical thresh. </string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
